src/libraries/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -686,6 +686,7 @@ public partial interface ITypeDescriptorContext : System.IServiceProvider
     }
     public partial interface ITypedList
     {
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Members of property types might be trimmed if not referenced directly")]
```

Comment:
We should add a period `.` on RUC messages.

Reply:
I initially did but they look weird when they display because trimmer adds a dot when it displays and it ends up showing two dots.. thoughts?

Reply:
I guess we should do a pass then on existing RUCs as I believe I intentionally remove them from a few places due to the duplicated period.